README.md

```diff
@@ -123,15 +123,15 @@ Google Cloud BigQuery (Alpha)
 
 Here is a code snippet showing a simple usage example from within Compute/App Engine. Note that you
 must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+Complete source code can be found at
+[CreateTableAndLoadData.java](./gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/snippets/CreateTableAndLoadData.java).
 
 ```java
 import com.google.gcloud.bigquery.BigQuery;
```

Comment:
Do you think we still need the import statements here if we have a link to the complete source code?  It'd make the sample shorter, but then again, perhaps the import statements have some value to readers.  I'm fine either way, I just thought I'd raise the question.